src/docs/product/dashboards/index.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Dashboards
+sidebar_order: 6
+redirect_from:
+  - /workflow/visibility/
+  - /workflow/dashboards/
+  - /product/error-monitoring/dashboards/
+description: "Sentry's Dashboards help you gain detailed perspectives of your application's health by providing filtering tools to navigate your issues across multiple projects in a single view."
+---
+
+Dashboards help you gain detailed perspectives of your application's health by providing visibility to your errors and transactions across multiple projects in a single view.
+
+Dashboards are comprised of one or more dashboard widgets. Each widget is a visualization of one or more [Discover queries](/product/discover-queries/).
+
+## Overall View of Application
+
+![Widgets visualizing events, errors by country, affected users, and handled vs unhandled issues.](dashboard.png)
+
+When visiting Dashboards for the first time, your organization is provided a set of pre-built widgets. You may edit them to suit your organization's needs.
+
+## Customizing widgets
+
+<Alert title="Note" level="info">
+  Available if your organization is on either the Business plan or the Trial plan.
```

Comment:
We likely need to clarify this, as we have multiple flavors of business plan @jeremybourque. 
```suggestion
  Available if your organization is on either the Business plan or the Trial plan.
```